src/codem/preprocessing/preprocess.py

```diff
@@ -408,6 +429,30 @@ def _create_dsm(self, resample: bool = True) -> None:
             self.nodata = data.nodata
             self.crs = data.crs
 
+            if not self.fnd and self.crs is not None and not self.crs.is_projected:
```

Comment:
🪢 🪢 🪢

Reply:
why tie a knot when you can tie a lot...

Reply:
Should be a bit better; although to accommodate mypy, still needed to keep one `and` in there.